src/modules/App/Pages/ChatPage/ChatPage.js

```diff
@@ -1,13 +1,19 @@
-import React from 'react'
-
+import React,{useState} from 'react'
+import TextInput from '../../../Components/InputComponent/InputComponent'
 const ChatPage = () => {
+  const [inputValue,setInputValue] = useState()
+  
+  const textHandle = (event) => {
+    setInputValue(event)
+  }
+  
   return (
     <div className='.container-fluid'>
       <div class="row justify-content-md-center">
 
         <div class="col-md-2">Left</div>
 
-        <div class="col-md-8">Right</div>
+        <div class="col-md-8"><TextInput title="input" onChange={(event)=> textHandle(event)}/></div>
```

Comment:
It should be inside component. no extra code should be accept chat PDP tag (component)